openedx/adg/lms/student/helpers.py

```diff
@@ -0,0 +1,163 @@
+"""
+Helpers for edx student app
+"""
+from django.conf import settings
+from django.contrib.auth.tokens import default_token_generator
+from django.urls import reverse
+from django.utils.http import int_to_base36
+
+from openedx.adg.common.lib.mandrill_client.client import MandrillClient
+from openedx.adg.common.lib.mandrill_client.email_data import EmailData
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+from openedx.core.djangolib.markup import Text
+from student.models import UserProfile
+
+
+def compose_and_send_activation_email(user, activation_key):
+    """
+    Prepare and send email for account activation
+
+    Arguments:
+        user (User): Django user object
+        activation_key (str): Activation key that will be send as email content
+
+    Returns:
+        None
+    """
+    user_profile = UserProfile.objects.get(user=user)
+    root_url = configuration_helpers.get_value('LMS_ROOT_URL', settings.LMS_ROOT_URL)
```

Comment:
This line is repeated on line 129. Let's move it to another function and call it here and in line 129.

Reply:
I believe it's not a good idea to introduce a whole new function just of the fact that some line is coming multiple times in a file or project. Usually when we write these sort of functions we don't actually use that in some other places hence results into multiple function with same functionality. If it would have some complex logic or not so complex than it would have make a lot more sense. but here we are already calling some function for the given values.
Hope that's answers your concern.